functions/handle-form-submit.js

```diff
@@ -0,0 +1,53 @@
+const axios = require('axios')
+const jsonp = require('jsonp-client')
+
+// Promisified jsonp client
+const fetchJsonp = url => {
+  return new Promise((resolve, reject) => {
+    const handler = (err, data) => err ? reject(err) : resolve(data)
+    jsonp(url, handler)
+  })
+}
+
+const verifyUrl = 'https://www.google.com/recaptcha/api/siteverify'
+
+// TODO: double check this Pardot url. Note that it's inentionally missing the last two segments
+// because the function is made to work with multiple forms.
+const submitUrl = 'https://go.replicated.com/l/948072'
```

Comment:
We keep the first part of the Pardot submission URL in the function - this is copied from replicated-www, we'll want to confirm that it's the correct one to use for this site as well. Note that the last two segments of the URL are stored client side and are intentionally left out here.